CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add liquidity options to the onboaring flow
+- Pay fixed channel opening fees instead of onchain transaction fees
+- Increased maximum channel size
+- Add minimum channel size
+- Split onboading and create invoice flow
+- Format amounts in input fields
```

Comment:
:wrench:

```suggestion
- Add liquidity options to the on-boarding flow.
- Pay fixed channel-opening fees instead of on-chain transaction fees.
- Increase maximum channel size to X.
- Add minimum channel size.
- Split on-boarding flow from create-invoice flow.
- Format amounts in input fields.
- Set coordinator leverage to 2?
```

Reply:
>  Increase maximum channel size to X.

This is configurable, I wouldn't put a number here.

> Set coordinator leverage to 2?

It's not really set to 2 its made configurable on the selected liquidity option.